declarations_site/catalog/views.py

```diff
@@ -267,33 +264,31 @@ def sitemap_general(request):
     for news in NewsPage.objects.live():
         urls.append(news.url)
 
-    search = Declaration.search().params(search_type="count")
+    search = Search(index=OLD_DECLARATION_INDEX).params(size=0)
     search.aggs.bucket(
-        'per_region', 'terms', field='general.post.region', size=0)
+        'per_region', 'terms', field='general.post.region.raw', size=30)
 
     for r in search.execute().aggregations.per_region.buckets:
         if r.key == "":
             continue
 
         urls.append(reverse("region", kwargs={"region_name": r.key}))
 
-        subsearch = Declaration.search()\
-            .filter(
-                Term(general__post__region=r.key) &
-                Not(Term(general__post__office='')))\
-            .params(search_type="count")
+        subsearch = Search(index=OLD_DECLARATION_INDEX)\
+            .query(Q('term', general__post__region__raw=r.key) & ~Q('term', general__post__office_raw=''))\
+            .params(size=0)
```

Comment:
Since this does aggregation later on, we don't actually want any documents in the "hits" part of the result — saves some resources on search and bandwidth.